### PR TITLE
Fix typos, orthography, grammer, and clarify the syntax of the `tags` tag

### PR DESCRIPTION
**Pull request by martin-rueegg**
## Description
- Fix typos, orthography, and grammar to improve readability.
- Clarify the syntax of the `tags` tag as it was ambiguous.

## Motivation and Context

I thought, all the tags needed to be separated by hyphens (`-`). Instead, tags needed to be separated by spaces (` `), and only multi-word tags are kept together with hyphens.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:

- [x] My code follows the code style of this repository.
- ~My change requires a change to documentation (this usually means the notes in the description of a package).~
- ~I have updated the documentation accordingly (this usually means the notes in the description of a package).~
- ~I have updated the package description and it is less than 4000 characters.~
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey-community/chocolatey-packages/blob/master/CONTRIBUTING.md)
- ~The added/modified package passed install/uninstall in the chocolatey test environment.]~
- ~The changes only affect a single package (not including meta package).~

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -282,14 +284,14 @@ Pull Requests that have changes requested, and/or is marked with the label [0 -
...
-Team doesn't think there is anything to fix here or that time required to fix it doesn't justify the benfit.
+Team doesn't think there is anything to fix here or that time required to fix it doesn't justify the benefit.
 - [Unresolved](https://github.com/chocolatey-community/chocolatey-packages/labels/Unresolved)
-The issue is unresolved - no feedback, no interest and it probably expired.
+The issue is unresolved - no feedback, no interest, and it probably expired.
 - [0 - _Triaging](https://github.com/chocolatey-community/chocolatey-packages/labels/0%20-%20_Triaging)
 Team is checking, confirming and measuring out the request.
 - [Pending closure](https://github.com/chocolatey-community/chocolatey-packages/labels/Pending%20closure)
-The issue is approaching end of issue life time. This is annoucement that issue will soon get closed in few weeks unless there is some new activity.
+The issue is approaching end of issue lifetime. This is announcement that issue will soon get closed in few weeks unless there is some new activity.
```
Comment (pauby) [inline CONTRIBUTING.md:295]:
```suggestion
The issue is approaching end of issue lifetime. This is an announcement that the issue will soon get closed in a few weeks, unless there is some new activity.
```

**Review by pauby — CHANGES_REQUESTED** [review]
Thank you very much for going through this and fixing up the docs. Just one small change and I'm happy to merge it!

File: CONTRIBUTING.md (hunk 1 above)
Comment (martin-rueegg) [reply CONTRIBUTING.md]: Thanks, @pauby. Perfect!

**Comment by martin-rueegg** [comment]
> Thank you very much for going through this and fixing up the docs. Just one small change and I'm happy to merge it!

Awesome, thank you. Do you want me to rebase it into one or two commits or are you happy with the three they've become now?

**Review by pauby — APPROVED** [review]

**Comment by pauby** [comment]
@martin-rueegg awesome work! Thanks for your contribution!

**Pull request merged**